SitumWayfinding/Classes/WayfindingLibrary/SitumMap.swift

```diff
@@ -70,17 +70,29 @@ public protocol SitumMap {
     func setOnMapReadyListener(listener: OnMapReadyListener?)
 
     /**
-     Select a poi in the map
+     Select a poi in the map centering view in the poi and showing the label (if any) on top of poi icon
      - parameters:
        - poi: the SITPOI you want to select
        - completion: callback called when operation complete either successfully or with an error
      */
     func selectPoi(poi: SITPOI, completion: @escaping (Result<Void, Error>) -> Void)
 
     /**
-     Navigate to a poi in the map
+     Navigate to a poi in the map. This will start positioning, calculate the route to destination, center view in the
+     location of the user and show instructions on how to reach that poi to the user
```

Comment:
Sería suficiente con engadir esta documentación para aclara como funciona a navegación?

Reply:
Que che parece algo como:
Start the navigation to a poi in the current building. This will:
* Start the positioining if needed
* Calculate and draw the route from the current user location to the poi.
* Provide the step-by-step instructions to reach the poi.